README.md

```diff
@@ -24,14 +15,118 @@ Its meetings and membership are largely focused on the direction, design, and wo
 
 The following subprojects are owned by the Working Group:
 
-{{
-
-* template-project
-  * Description: Brief description of project. Remove this item and add new projects using this format.
+* Communication
+  * Description: Packages for underwater network wireless communication.
+  * Motive: Since underwater communications relies on acoustic waves, there are some specificities regarding these types of waves that should be taken into account during data transmission.
   * Repositories
-    * link-to-repository
-
-}}
+    * Suggestions...
+  * References: 
+    *  [ns uan](https://www.nsnam.org/docs/models/html/uan.html)
+
+* Common Messages
+  * Description: Specifics ROS messages for maritime robotics.
+  * Motive: There are maritime robotics sensors that gives specific type of messages.
+  * Unique Repo
+    *  Assist the development of [UW APL Hydrographic Messages](https://github.com/apl-ocean-engineering/hydrographic_msgs)
+
+* Drivers
+  * Description: Documentation and index for underwater sensors and actuators drivers.
+  * Motive: There are specifics components used only for maritime robotics, which requires drivers to interact with them. Although, there already is an ROS organization to develop ROS drivers, so instead of developing inside this organization, we would use it just to have an index describing maritimate focused ROS drivers.
+  * Unique Repo
+    * **[Proposal]** ROS Maritime Robotics Drivers Index
+    * **[Proposal]** Sonar and radar drivers
```

Comment:
But how would that change the charter?  What I meant by that is that we have a repo only for indexing the available drivers and to also code drivers in case needed.